lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class Car
+  class Car
+    attr_reader :color, :year, :model, :current_speed
+
+    def initialize(car_attributes)
+      @color = car_attributes.fetch(:color, 'black')
+      @year  = car_attributes.fetch(:year, 2017)
+      @model = car_attributes.fetch(:model, 'tesla_model_3')
+      @current_speed = nil
+    end
+
+    def self.default_car
+      Car.new
+    end
+
+    def speed_up(speed)
+      @current_speed = speed
```

Comment:
if I try increase - I have error
` 1) Error:
Car::#speed_up#test_0002_returns full info:
NoMethodError: undefined method `+' for nil:NilClass
    /home/harusan/RubymineProjects/homework2/lib/topics/state_and_behaviour.rb:25:in `speed_up'
    /home/harusan/RubymineProjects/homework2/test/topics/test_state_and_behaviour.rb:79:in `block (3 levels) in <class:StateAndBehaviourTest>'`